internal/iostreams/iostreams_mock.go

```diff
@@ -98,13 +98,13 @@ func (m *IOStreamsMock) IsTTY() bool {
 }
 
 // SetExitCode sets the desired exit code in a thread-safe way
-func (io *IOStreamsMock) SetExitCode(code ExitCode) {
-	atomic.StoreInt32((*int32)(&io.exitCode), int32(code))
+func (m *IOStreamsMock) SetExitCode(code ExitCode) {
```

Comment:
🧪 Nice! I like `mock` being `m` if this is what's found elsewhere!

Reply:
Yea, `m` seems to be a common receiver name for mocks. Makes sense to me and keeps it consistent.